src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIChatClient.cs

```diff
@@ -627,11 +627,10 @@ private static ChatCompletionOptions ToOpenAIOptions(ChatOptions? options)
     /// <summary>Converts an Extensions function to an OpenAI chat tool.</summary>
     private static ChatTool ToOpenAIChatTool(AIFunction aiFunction)
     {
-        // Default strict to true, but allow to be overridden by an additional Strict property.
-        bool strict =
-            !aiFunction.AdditionalProperties.TryGetValue("Strict", out object? strictObj) ||
-            strictObj is not bool strictValue ||
-            strictValue;
+        bool? strict =
+            aiFunction.AdditionalProperties.TryGetValue("Strict", out object? strictObj) &&
+            strictObj is bool strictValue ?
+            strictValue : null;
```

Comment:
There are some other places that turn on strict as well:
https://github.com/search?q=repo%3Adotnet%2Fextensions+%22strict%3A+true%22&type=code

Reply:
See https://github.com/dotnet/extensions/pull/6064 for the changes that were made to turn it on

Reply:
Incorporated.